src/lib/components/forms/form/helper/terra-form-field.helper.ts

```diff
@@ -221,7 +221,8 @@ export class TerraFormFieldHelper {
         let max: number;
 
         if (isString(range)) {
-            let match: RegExpExecArray = /^\[(\d*),(\d*)]$/.exec(range as string);
+            range = (range as string).replace(/\s/g, '');
+            let match: RegExpExecArray = /^\[(\d*)\s*,\s*(\d*)]$/.exec(range as string);
```

Comment:
Lieber eine neue Variable erstellen um Seiteneffekte zu vermeiden
```suggestion
            const trimmedRange:string = (range as string).replace(/\s/g, '');
            let match: RegExpExecArray = /^\[(\d*)\s*,\s*(\d*)]$/.exec(trimmedRange as string);
```